src/Contracts/DataModels.ts

```diff
@@ -450,6 +450,15 @@ export interface IResponse<T> {
   data: T;
 }
 
+export interface IValidationError {
+  message: string;
+  type: string;
+}
+
+export interface IMaxAllocationTimeExceeded extends IValidationError {
```

Comment:
more data was added for too many users per dbaccount and to many accounts per user, pls add them too